fixtures/cli_input/src/other_dir/other_file.rs

```diff
@@ -0,0 +1,20 @@
+#[test]
+fn other_file_test() {
+    let mut n = 0;
+    n += 1;
+    other_file_noop();
+}
+
+#[allow(dead_code)]
+fn other_file_noop() {}
+
+#[test]
+fn unused_other_test() {
+    let value = unused_other_expression();
+    assert!(value);
+}
+
+#[allow(dead_code)]
+fn unused_other_expression() -> bool {
+    true
+} 
```

Comment:
```suggestion
```
Did you have a specific reason for including these? If not, let's please remove them, just to keep things simple.